python/mujinplanningclient/binpickingplanningclient.py

```diff
@@ -46,641 +75,2936 @@ def __init__(self, regionname=None, **kwargs):
+            deletetarget: whether to delete target after pick and place is done (Default: 0)
+            debuglevel: Sets the debug level for the planning logs. For development. 3=INFO, 4=DEBUG, 5=VERBOSE. (Default: 4)
+            movetodestination: planning parameter (Default: 1)
+            freeinc: planning parameter (Default: [0.08])
+            worksteplength: **deprecated** planning parameter (Default: None)
+            densowavearmgroup: planning parameter (Default: 5)
+            regionname: Name of the region of the objects. (Default: None)
+            cameranames: The names of the cameras to avoid occlusions with the robot (Default: None)
+            envclearance: Environment clearance in millimeters. (Default: None)
+            toolname: Name of the manipulator. Defaults to currently selected tool (Default: None)
+            robotspeed: Value in (0,1] defining the percentage of speed the robot should move at. (Default: None)
+            timeout: Time in seconds after which the command is assumed to have failed. (Default: 1000)
+            leaveoffsetintool: If 1, destdepartoffsetdir is in the tool coordinate system. If 0, destdepartoffsetdir is in the global coordinate system. By default this is 0. (Default: None)
+            desttargetname: The destination target name where the destination goal ikparams come from (Default: None)
+            destikparamnames: A list of lists of ikparam names for the destinations of the target. Only destikparamnames[0] is looked at and tells the system to place the part in any of the ikparams in destikparamnames[0] (Default: None)
+            graspsetname: The name of the grasp set that belongs that should be used to grasp the target. (Default: None)
+            dynamicEnvironmentState: Dynamic environment state that allows the user to set/create objects in a particular state dynamically. (Default: None)
+        """
         assert (targetnamepattern is not None)
         if regionname is None:
             regionname = self.regionname
         taskparameters = {
             'command': 'PickAndPlace',
-            'goaltype': goaltype,
-            'envclearance': envclearance,
-            'movetodestination': movetodestination,
-            'goals': goals,
+            'targetnamepattern': targetnamepattern,
             'approachoffset': approachoffset,
             'departoffsetdir': departoffsetdir,
             'destdepartoffsetdir': destdepartoffsetdir,
-            'freeinc': freeinc,
-            'worksteplength': worksteplength,
-            'targetnamepattern': targetnamepattern,
             'deletetarget': deletetarget,
             'debuglevel': debuglevel,
-        }
+            'movetodestination': movetodestination,
+            'freeinc': freeinc,
+            'envclearance': envclearance,
+        }  # type: dict[str, Any]
+        if densowavearmgroup != 5:
```

Comment:
I think this is a behavior change, even though I don't think this parameter is used anywhere at all

Reply:
https://github.com/mujin/mujinplanningclientpy/pull/45/commits/12c1774484ac38e12a9c4b7b8fd02a5fde53f85b -- I just got rid of the parameter since it's only referenced in tests.